### DISCUSSION TITLE
When will Copilot chat support IntelliJ IDEA?

### DISCUSSION THREAD

**Discussion by luoxiaofeng1029** [community/community#58504] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

When will Copilot chat support IntelliJ IDEA?

**Comment by Akash1134** [comment]
Hi @luoxiaofeng1029 ,

Thank you for posting in the Copilot community! We value your feedback and our team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions.

At this time, the public preview for Copilot for Chat is for VS or VSCode users and we do not have an ETA for supporting other IDEs right now but we'll update you when we have details to share as we're working on it. We appreciate your patience and understanding.

**Reply by rnevet-reply** [reply]
@Akash1134 what about now?

**Reply by 1Bugivugi** [reply]
@Akash1134 ![image](https://github.com/community/community/assets/33460185/4f689baf-034e-4b20-9f3f-67823a3c2b28)

**Reply by fjmn2001** [reply]
Something new about it? 🤔

**Reply by oliver-mentel** [reply]
Any news on this? 🙏🏼

**Reply by HonzaTuron** [reply]
https://twitter.com/LinusEkenstam/status/1722320525454676063 Should be supported soon.

**Comment by lobner** [comment]
Even better yet:
https://github.com/orgs/community/discussions/50808#discussioncomment-7514374


ons. 8. nov. 2023 22.20 skrev Jan Turoň ***@***.***>:

**Comment by Akash1134** [comment]
👋 We are happy to announce that a private beta of GitHub Copilot Chat is now available for users of JetBrains IDEs, including IntelliJ, PyCharm, WebStorm, Android Studio, and more. 🚀

This private beta is available to Copilot Business customers and Copilot Individual users.

To get access to the private beta, [sign up for this waitlist](https://github.com/github-copilot/chat_jetbrains_waitlist_signup/join).
GitHub Enterprise Cloud (GHEC) administrators interested in participating in the private beta should reach out to your GitHub account manager or [contact our sales team](https://github.com/enterprise/contact) to make the feature available for your enterprise.

We’d love your feedback on this new release. Please [use this link to share your feedback](https://gh.io/copilot-chat-jb-feedback) or ideas on how to improve the product. Thank you for your patience and support.

**Reply by chathurabuddi** [reply]
Even if I enable the chat feature from the organization setting, it asks to join the waitlist in Intellij Idea and can't really use it :(

**Reply by ghost** [reply]
Yes, we have the same issue.

**Reply by fjmn2001** [reply]
I have the same issue

**Reply by eduardomanrique** [reply]
Same issue

**Reply by CarlosBorroto** [reply]
Same. I'm part of an organization with the Copilot Chat feature enabled. I confirmed I have access to the tool using VS Code. In PyCharm, I see this message instead.

<img width="606" alt="Screenshot 2023-11-13 at 6 55 42 AM" src="https://github.com/community/community/assets/240588/75cc5118-c486-426b-9d27-35cda70c4efe">

**Reply by DOS-SOCOMEC** [reply]
Same problem.

**Reply by HAZEM-MEJRI** [reply]
any solutions / updates ?

**Reply by jakobww** [reply]
same issue here

**Reply by vetler** [reply]
Here as well 👋

**Reply by josessbcp** [reply]
Same problem

**Reply by diegopamio** [reply]
Same here.

**Reply by razvanbugariu** [reply]
Same here

**Reply by DerAzubi** [reply]
Same here. Even with the newest Intellij and Plugin Versions

**Reply by ghost** [reply]
any solutions so far? its quite annoying

**Reply by wimvanhooff-mtv** [reply]
Still not working

**Reply by timtilch** [reply]
Same here

**Reply by arashgl** [reply]
same here ,its very annoying that we can't use copilot chat!

**Reply by clintonbrito** [reply]
The best part of using Copilot is having the chat to inquire about the code and debug. Any news about that? I've already signed for the waitlist, but no response yet.

**Reply by BeCaDRI** [reply]
feels like a ever lasting Early Access game on steam

**Comment by Akash1134** [comment]
👋 Hi everyone, The waitlist is open to Copilot Individuals users. Copilot Business invoiced enterprise customers interested in participating in the private beta should reach out to your GitHub account manager or [contact our sales team](https://github.com/enterprise/contact) to make the feature available for your enterprise. Customers on Free, Team or self-serve Enterprise plans do not have an account manager and are not eligible for the private beta at this time. Thanks!

**Comment by trevorius** [comment]
can we use this feature if we use it in vscode already or do we have to join the waitlist anyway ?

**Comment by DOS-SOCOMEC** [comment]
I found the problem : https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide
"If you are an owner of an invoiced organization or enterprise with a Copilot Business subscription and would like to participate in the beta for GitHub Copilot Chat in a JetBrains IDE, you can contact your account manager or solutions engineer to request access. **Customers on Free, Team or self-serve Enterprise plans do not have an account manager and are not eligible for the private beta at this time**."

**Reply by DOS-SOCOMEC** [reply]
I also received an additional response during discussions with GitHub salespeople.

"Unfortunately paying for Enterprise right now wouldn't even guarantee that you're accepted to the private preview, but that's what makes you eligible to apply."

Basically, if you weren't already a Github "Enterprise" customer, you probably can't access "Chat" in IntellijIdea... Really disappointed with this restriction that's between the lines.

And for Github Enterprise customers, it seems you just need to contact your account manager. (This is not the admin of your org but the github sales contact of your org)

**Comment by trevorius** [comment]
the copilot pluggin for webstorm has got a chat verion in it now.

**Comment by mwesarg** [comment]
When will the private beta be over?

**Comment by DreamsBR** [comment]
Wait per a service that we are paying per two or more months...

**Comment by Akash1134** [comment, marked as answer]
Following our [Private Beta](https://github.blog/changelog/2023-11-08-copilot-in-jetbrains-ides-chat-is-now-available-in-private-beta-new-welcome-guide/), we are thrilled to announce **Copilot Chat in JetBrains IDEs is now generally available (GA)** 🎉

Driven by GPT-4, GitHub Copilot Chat provides instant guidance directly within various JetBrains IDEs, such as PyCharm, IntelliJ IDEA, WebStorm, Rider, and more. This contextually-aware tool tailors suggestions to your specific coding tasks and even allows explicitly adding files for reference. It empowers developers to innovate efficiently by assisting with complex concepts, code explanations, unit testing, and many more use cases, all while effortlessly adjusting to your preferred language style. 

You can learn more about the release in this [changelog](https://github.blog/changelog/2024-03-07-github-copilot-chat-general-availability-in-jetbrains-ide/). 

If you've got any questions or feedback to share, please post them in [this discussion](https://github.com/orgs/community/discussions/110614).
